### ISSUE TITLE
[CoE Starter Kit - QUESTION] Cloud Flows in BYODL Architecture

### ISSUE THREAD

**Issue by DominicMeznik** [microsoft/coe-starter-kit#8487]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

Is there a list of (inventory) Cloud Flows that still need to be activated after switching to BYODL? I guess some data still needs to be collected by cloud flows and not everything can be exported from the Datalake.

I wanted to cross check, if everything is activated as it should be

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

June 2024

### What method are you using to get inventory and telemetry?

Data Export

AB#3603

**Comment by Jenefer-Monroe** [comment]
Hello, if you use the setup wizard it will ensure all the correct flows are turned on depending on the architecture you chose.
This information is also stored in the CoE Solution Metadata table:
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/3a885009-45e1-46db-8cac-f91d6bfcb058)

### Use the setup wizard
Its highly recommended that you use the Setup Wizard to walk you through setup and upgrades. It will ensure all the new flows are turned on in the correct order, trigger things like population of the bookmarks screen and do other cleanup steps needed.
You should do this both for upgrades and clean installs.
To use it, first perform the import of the solution install or upgrade manually, then boot the app and walk through the screens.
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/b84cc1ff-383a-497c-9236-569e8313c73c)
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/ed02f0d0-c276-48cc-90af-47e5bf5a928e)

Please do let me know if I"ve answered your question.

**Comment by DominicMeznik** [comment]
Thanks for your answer! I was just wondering, since after switching over (not starting with a clean Environment) the main inventory flows were still on and I turned them off manually, so I just wanted to cross check, which one´s are actually required.

**Comment by Jenefer-Monroe** [comment]
oh I see. Did you use the setup wizard? As I recall, we did add a little flow which would turn off the no longer needed flows. 
But if that did not run you can instead use the setup wizard to see which ones are not needed for Data Export and turn those off. 
Thanks!

**Comment by DominicMeznik** [comment]
Yes, I did it with the setup wizard. Maybe I did something wrong

**Comment by Jenefer-Monroe** [comment]
OK no worries as we move into the big Data Export feature work we've got ahead I will revisit that. 
Hopefully you are unblocked now though.

**Comment by DominicMeznik** [comment]
Yes, with the provided information everything is good now, thank you very much!
